Store Credentials in database (#2)
magma-labs/solidus_mercado_pago: README.md

This will import assets and migrations

> Basi Setup

Reviewer: ```suggestion
Basic setup
```

Author: Done

-----

## Retrieve Mercado pago account details
You'll need the following account details:

- Client ID
- Client secret

> - Sandbox

Reviewer: ```suggestion
- Sandbox (A boolean value to indicate if we're using the sandbox or not, by default `true`)
```

Author: Done!

These values can be obtained by logging in to your Mercado Pago account, going to `Credentials -> Basic checkout`

## Create a new payment method

Payment methods can accept preferences either directly entered in admin, or from static source in code. For most projects we recommend using a static source, so that sensitive account credentials are not stored in the database.

> 1. Set static preferences in an initializer

Reviewer: ```suggestion
1. Set static preferences in an initializer:
```

Author: This option was removed

```ruby
# config/initializers/spree.rb
Spree::Config.configure do |config|
  config.static_model_preferences.add(
    Spree::PaymentMethod::MercadoPago,
    'mercado_pago_credentials', {
      sandbox: Rails.env.production? ? true : false,
      client_id: ENV['MERCADOPAGO_CLIENT_ID'],
      client_secret: ENV['MERCADOPAGO_CLIENT_SECRET']
    }
  )
end
```

> Other option is create the payment method from admin interface:

Reviewer: ```suggestion
2. Another option is to create the payment method from admin interface:
```

Author: This section was removed

1. Visit `/admin/payment_methods/new`

> 2. Set as provider the `Mercado pago` option

Reviewer: ```suggestion
2. Set `Mercado pago` as provider
```

Author: Done!

> 3. Save to refresh the form

Reviewer: ```suggestion
3. Save to update the form
```

Author: Done!
